holonote/annotate/plotting.py

```diff
@@ -0,0 +1,505 @@
+from __future__ import annotations
+
+import weakref
+from typing import TYPE_CHECKING, Any
+
+import holoviews as hv
+import pandas as pd
+import param
+from bokeh.models.tools import BoxSelectTool, HoverTool, Tool
+
+from .._warnings import warn
+
+if TYPE_CHECKING:
+    from .annotator import Annotator
+
+
+class _StyleOpts(param.Dict):
+    def _validate(self, val) -> None:
+        super()._validate(val)
+        for k in val:
+            if k in ("color", "alpha"):
+                warn("Color and alpha opts should be set directly on the style object.")
+
+
+_default_opts = {"apply_ranges": False, "show_legend": False}
+
+
+class Style(param.Parameterized):
+    alpha = param.Number(default=0.2, bounds=(0, 1), allow_refs=True)
+    alpha.doc = "Alpha value for non-selected regions"
```

Comment:
Setting the `doc` this way is not consistent/idiomatic param usage - it should be a keyword argument in the Parameter declaration.

Reply:
Will fix